books/free-programming-books-langs.md

```diff
@@ -427,7 +427,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Beej's Guide to the GNU Debugger (GDB)](http://beej.us/guide/bggdb/) - B. Hall (HTML)
 * [Build Your Own Lisp](http://www.buildyourownlisp.com) - Daniel Holden
 * [C Elements of Style](http://www.oualline.com/books.free/style/) - Steve Oualline
-* [C for Python Programmers - Carl Burch](http://www.toves.org/books/cpy/)
+* [C for Python Programmers - Carl Burch](http://www.cburch.com/books/cpy/)
```

Comment:
can we move the author attribution to the right place, too?

Reply:
Yes. Will do that.